src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -188,6 +199,33 @@ def _get_rop_steps(rop: hou.Node):
     return rop_steps
 
 
+# Any changes to this function should be reflected in the user guide.
+def _get_render_strategy_for_node(node: hou.Node) -> RenderStrategy:
+    render_strategy = RenderStrategy.PARALLEL
+
+    if (
+        node.type().nameWithCategory() == "Driver/geometry"
+        and node.parm("initsim")
+        and node.parm("initsim").eval()
+    ):
+        render_strategy = RenderStrategy.SEQUENTIAL
+
+    if node.parm("deadline_cloud_render_strategy"):
+        strategy_string = node.parm("deadline_cloud_render_strategy").evalAsString()
+        if strategy_string.upper() == RenderStrategy.SEQUENTIAL.value:
+            render_strategy = RenderStrategy.SEQUENTIAL
+        elif strategy_string.upper() == RenderStrategy.PARALLEL.value:
+            render_strategy = RenderStrategy.PARALLEL
+        elif strategy_string == "":
+            # Use the submitter's default choice
+            pass
+        else:
+            raise Exception(
+                f'node {node.path()} has an unexpected value "{strategy_string}" for its "deadline_cloud_render_strategy" parameter. Choose either {RenderStrategy.PARALLEL.value} or {RenderStrategy.SEQUENTIAL.value}'
+            )
```

Comment:
Have we tested how this error shows up to the user to ensure it matches the usability we want?

Reply:
It pops up an error message that looks like this: <img width="513" alt="Screenshot 2024-05-22 at 7 31 34 AM" src="https://github.com/aws-deadline/deadline-cloud-for-houdini/assets/6042774/e0c4ba5d-ec97-4f5a-8853-8fbc4bd2275a">

Reply:
1. In my above comment, I changed this to a `ValueError` instead of the generic `Exception`, can we do that here regardless of the above conversation?

2. nit: Maybe clean up the language a bit like:

```suggestion
            raise ValueError(
                f'The node "{node.path()}" has an unexpected value "{strategy_string}" for its "deadline_cloud_render_strategy" parameter. Ensure the value is {RenderStrategy.PARALLEL.value} or {RenderStrategy.SEQUENTIAL.value}'
            )
```

Not sure I want to add quotes around the enum values in-case a user accidentally adds quotes to the values.